src/app/home/home.component.ts

```diff
@@ -16,10 +15,10 @@ import * as serviceOfferingActions from '../reducers/service-offerings/redux/ser
 })
 export class HomeComponent extends WithUnsubscribe() implements OnInit {
   public disableSecurityGroups = false;
+  public showSidenav$ = this.store.select(layoutSelectors.getIsShowSidenav);
```

Comment:
why do you use this language?
"is show sidenav" why not to use" "is sidenav visible"